src/context/delta.rs

```diff
@@ -333,15 +333,32 @@ impl SeafowlContext {
                 let table_log_store = self
                     .internal_object_store
                     .get_log_store(&table_uuid.to_string());
-                let table = ConvertToDeltaBuilder::new()
-                    .with_log_store(table_log_store)
+
+                let table = match ConvertToDeltaBuilder::new()
+                    .with_log_store(table_log_store.clone())
                     .with_table_name(&*table_name)
                     .with_save_mode(SaveMode::Overwrite)
                     .with_comment(format!(
                         "Converted by Seafowl {}",
                         env!("CARGO_PKG_VERSION")
                     ))
-                    .await?;
+                    .await
+                {
+                    Ok(table) => table,
+                    // Match on table already exist error
```

Comment:
(to avoid some WTFs)

```suggestion
                    // Match on table already exist error
                    // We can't match on the enum value directly, since this
                    // Error enum isn't public
```